FeatureGroupOverrides.json

```diff
@@ -0,0 +1,18 @@
+{
+  "calls": {},
+  "apis": {},
+  "subgroups": {
+    "Account Management": "AccountManagement",
+    "Characters": "Character",
+    "Character Data": "Character",
+    "Server-Side Cloud Script": "CloudScript",
+    "Friend List Management": "Friends",
+    "Custom Server Management": "GameServer",
+    "Platform Specific Methods": "PlatformSpecificMethods",

```

Comment:
>PlatformSpecificMethods [](http://example.com/codeflow?start=34&length=23)

"PlatformSpecific" is likely better.  adding Methods is kinda odd compared to other naming

Reply:
Yes, I was going to suggest exactly that. The category should just be "PlatformSpecific"